cpp/log/log_signer.cc

```diff
@@ -252,34 +248,34 @@ LogSigVerifier::VerifyResult LogSigVerifier::VerifySTHSignature(
   if (sth.id().has_key_id() && sth.id().key_id() != KeyID())
     return KEY_ID_MISMATCH;
   string serialized_sth;
-  Serializer::SerializeResult serialize_result =
+  SerializeResult serialize_result =
       Serializer::SerializeSTHSignatureInput(sth, &serialized_sth);
-  if (serialize_result != Serializer::OK)
+  if (serialize_result != SerializeResult::OK)
     return GetSerializeError(serialize_result);
   return ConvertStatus(Verify(serialized_sth, sth.signature()));
 }
 
 // static
 LogSigVerifier::VerifyResult LogSigVerifier::GetSerializeError(
-    Serializer::SerializeResult result) {
+    SerializeResult result) {
   VerifyResult verify_result;
   switch (result) {
-    case Serializer::INVALID_ENTRY_TYPE:
+    case SerializeResult::INVALID_ENTRY_TYPE:
```

Comment:
Again I think we have a proliferation of enums that represent very similar things. It's OK for now.